### ISSUE TITLE
why freeze img backbone in both stage?

### ISSUE THREAD

**Issue by buaazeus** [OpenDriveLab/UniAD#155]
Could you please explain why img backbone is freezed both in stage1 and stage2? The img backbone weights are pretrained and not updated in UniAD training process.
Thank you.

**Comment by ilnehc** [comment]
@buaazeus Thanks for the question. Training backbones with the tracking module usually need to save gradients for all timestamps, which will cause OOM.
